### ISSUE TITLE
Pagination component should allow "no item founds" message

### ISSUE THREAD

**Issue by rafasantos** [rafasantos/matchandtrade-web-ui#2]
## Description
Pagination component should allow custom message when there are no items to be displayed.

## Proposed Usage
```
<app-pagination
	[notFoundMessage]="Sorry but there are no available trades at the moment."
	[loading]="loading"
	[pagination]="pagination"
	(onNextPage)="nextPage()"
	(onPreviousPage)="previousPage()">
</app-pagination>
```

**Comment by rafasantos** [comment]
fix 9158f76f54dcc10056364a379926cf9e43429382

**Comment by rafasantos** [comment]
fix #9158f76f54dcc10056364a379926cf9e43429382
